gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -213,6 +213,10 @@
   public static final HadoopConfigurationProperty<Long> GCS_REWRITE_MAX_BYTES_PER_CALL =
       new HadoopConfigurationProperty<>("fs.gs.rewrite.max.bytes.per.call", 512 * 1024 * 1024L);
 
+  /** Configuration key for enabling gcs debug logging. */
```

Comment:
nit: Does the description explain anything that the value's name itself does?

Reply:
Added the logged package info in the comment.